### ISSUE TITLE
Shapes of retouch module are not in the mask manager module

### ISSUE THREAD

**Issue by pahervin** [darktable-org/darktable#10336]
**Describe the bug/issue**
When editing complex retouches, it becomes hard to select a shape among tens. Shapes are not displayed in the mask module.

**To Reproduce**
1. Edit a photo with the retouch module
2. Activate the healing tool or the cloning tool
3. Edit several retouches with a shape (circle, ellipse or path)

**Expected behavior**
The retouch module shapes are viewable/selectable  from the maskmanager module

**Screenshots**
![masques-de-retouche](https://user-images.githubusercontent.com/6032499/140505910-55239caf-e443-4202-a348-de68d0166f0d.jpg)

**Platform**
* darktable version : 3.6.1
* OS                : Win10 (21H1)
* Memory            : 8GB
* Graphics card     : Intel UHD Graphics 620 and Radeon 520
* Graphics driver   : 
* OpenCL installed  :  no

**Additional context**
 - Can you reproduce with another darktable version(s)? no
 - Do you use lua scripts? no

**Comment by TurboGit** [comment]
That's expected. The forms in Retouch module (and it was also the case for the now deprecated spot removal) are not part of the mask manager. It is also the case for the Liquify module BTW (even if more comprehensible as very different visually).

**Comment by pahervin** [comment]
> 
> 
> That's expected. The forms in Retouch module (and it was also the case for the now deprecated spot removal) are not part of the mask manager. It is also the case for the Liquify module BTW (even if more comprehensible as very different visually).

Ok.
In the given example, source shapes overlay often, and it is very touchy to select the one I want to edit.
Is there a way to select a retouch shape from a list ? Within the module itself  or elsewhere ?

**Comment by TurboGit** [comment]
> In the given example, source shapes overlay often, and it is very touchy to select the one I want to edit.

This is fixed in master where fully overlapping shapes can be selected.

**Comment by ruudsieb** [comment]
What is the reason of the retouch masks not being available in the mask manager? I think it would benefit usability if all masks would be treated equal and would be available on a central location like the mask manager.
My use case (when I stumbled upon this): I added a fill color mask in the retouch module and wanted to add grain to the same area to let it blend in better. I had to redraw the mask in the grain module. Reusing a mask via the maskmanager would've been much easier.
